node-lib/src/config_files/rpc.rs

```diff
@@ -53,30 +53,30 @@ impl RpcConfigFile {
         config_file: RpcConfigFile,
         options: &RunOptions,
     ) -> RpcConfigFile {
-        let default_http_rpc_addr = Self::default_bind_address(chain_config);
+        let default_rpc_bind_address = Self::default_bind_address(chain_config);
 
         let RpcConfigFile {
-            http_bind_address,
-            http_enabled,
+            bind_address,
+            rpc_enabled: http_enabled,
```

Comment:
Makes sense to rename this `http_enabled` too (and the one below)

Reply:
Fixed in another PR. Sorry I missed this.